src/run-command.js

```diff
@@ -15,5 +15,5 @@ export default async function runCommand(command, args) {
 		version,
 	});
 
-	return execa(`${directory}/src/${command}`, args, { stdio: 'inherit' });
+	return { childProcess: execa(`${directory}/src/${command}`, args, opts) };
```

Comment:
```suggestion
	return execa(`${directory}/src/${command}`, args, { stdio, detached });
```

I'd prefer to control the options explicitly rather than allowing the caller to pass it anything.

Also, all of the CLI commands (redis-benchmark, redis-server, etc) expect this to have `execa`s return value, aka a promise), so they'll all break. I don't currently have tests for those files, which is why it isn't currently failing, but they should be.